ingestion/src/metadata/ingestion/source/pipeline/fivetran/metadata.py

```diff
@@ -100,25 +103,74 @@ def yield_pipeline(
         :param pipeline_details: pipeline_details object from fivetran
         :return: Create Pipeline request with tasks
         """
+        source_url = self.get_source_url(
+            connector_id=pipeline_details.source.get("id"),
+            group_id=pipeline_details.group.get("id"),
+            source_name=pipeline_details.source.get("service"),
+        )
         pipeline_request = CreatePipelineRequest(
             name=EntityName(pipeline_details.pipeline_name),
             displayName=pipeline_details.pipeline_display_name,
-            tasks=self.get_connections_jobs(pipeline_details),
-            service=FullyQualifiedEntityName(self.context.get().pipeline_service),
-            sourceUrl=self.get_source_url(
-                connector_id=pipeline_details.source.get("id"),
-                group_id=pipeline_details.group.get("id"),
-                source_name=pipeline_details.source.get("service"),
+            tasks=self.get_connections_jobs(
+                pipeline_details=pipeline_details, source_url=source_url
             ),
-        )
-        yield Either(right=pipeline_request)
+            service=FullyQualifiedEntityName(self.context.get().pipeline_service),
+            sourceUrl=source_url,
+        )  # type: ignore
+        yield Either(left=None, right=pipeline_request)
         self.register_record(pipeline_request=pipeline_request)
 
     def yield_pipeline_status(
         self, pipeline_details: FivetranPipelineDetails
-    ) -> Iterable[Either[OMetaPipelineStatus]]:
+    ) -> Optional[Iterable[Either[OMetaPipelineStatus]]]:
         """Method to get task & pipeline status"""
 
+    def fetch_column_lineage(
+        self, pipeline_details: FivetranPipelineDetails, schema, schema_data, table
+    ):
```

Comment:
should we add a return type here 
`Optional[List[ColumnLineage]]`

Reply:
Thanks, I have added the relevant return typehint.